tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -1145,3 +1159,40 @@ def _test_recover_from_protected_mode(
     # Test after pcluster stop and then start, static nodes are not treated as bootstrap failure nodes,
     # not enter protected mode
     check_status(cluster, compute_fleet_status="RUNNING")
+
+
+def _test_compute_node_bootstrap_timeout(
+    cluster,
+    pcluster_config_reader,
+    remote_command_executor,
+    compute_node_bootstrap_timeout,
+    scaledown_idletime,
+    gpu_instance_type,
+):
+    """Test compute_node_bootstrap_timeout is passed into slurm.conf and parallelcluster_clustermgtd.conf."""
+    slurm_parallelcluster_conf = remote_command_executor.run_remote_command(
+        "sudo cat /opt/slurm/etc/slurm_parallelcluster.conf"
+    ).stdout
+    assert_that(slurm_parallelcluster_conf).contains(f"ResumeTimeout={compute_node_bootstrap_timeout}")
+    clustermgtd_conf = remote_command_executor.run_remote_command(
+        "sudo cat /etc/parallelcluster/slurm_plugin/parallelcluster_clustermgtd.conf"
+    ).stdout
+    assert_that(clustermgtd_conf).contains(f"node_replacement_timeout = {compute_node_bootstrap_timeout}")
+    # Update cluster
+    update_compute_node_bootstrap_timeout = 1200
+    updated_config_file = pcluster_config_reader(
+        scaledown_idletime=scaledown_idletime,
+        gpu_instance_type=gpu_instance_type,
+        compute_node_bootstrap_timeout=update_compute_node_bootstrap_timeout,
+        config_file="pcluster.update.config.yaml",
```

Comment:
do we really need a different file or we can re-render the same with a different `compute_node_bootstrap_timeout`? the advantage is that we don't have 2 files to keep in sync

Reply:
Had been try using the same config file and render with different compute_node_bootstrap_timeout. However it doesn't work and use the previous rendeed result.

Reply:
You are saying that if you do something like:
```
updated_config_file = pcluster_config_reader(
        scaledown_idletime=scaledown_idletime,
        gpu_instance_type=gpu_instance_type,
        compute_node_bootstrap_timeout=update_compute_node_bootstrap_timeout,
)
```
you don't get the correct config?